site/docs/v1/tech/shared/_data_field_data_type_changes.adoc

```diff
@@ -0,0 +1,8 @@
+FusionAuth provides `data` fields on the `user` and `registration` objects. You should not modify the data type of fields stored in `data`. 
```

Comment:
We actually add this `data` field to lots of objects

- Applications
- Tenants
- Groups
- Users
- Registrations
- Consent

Others probably... so in theory they should use the same discipline in these as well. The main issue with User and Registration data is that it is indexed in Elasticsearch. 

But if we were to ever index other things the same issue would arise. So perhaps we should provide a word of caution anytime they use the `.data` field.

Reply:
Thanks, great points.